docpad.coffee

```diff
@@ -11,21 +11,21 @@ module.exports =
     # Conference info
     conf:
       name: 'Vale dos Desenvolvedores'
-      description: 'Valley of Developers Meetup Roadmap 2017'
-      date: '18/Nov de 2017'
+      description: 'Valley of Developers'
+      date: '18/Abril de 2018'
```

Comment:
I suppose that this date is wrong. Isn't Abril 14?

Reply:
Yeah, It is.